### DISCUSSION TITLE
Backup from CephFS to S3 never work consistently

### DISCUSSION THREAD

**Discussion by ccahyadi-lyrid** [vmware-tanzu/velero#7916] [Community support Q&A]
![image](https://github.com/vmware-tanzu/velero/assets/139726789/7c1dfcd8-6ccc-49b7-84c2-aca76ab9f04c)
data upload seems never complete successfully,
From what i can see, 
The log says this
time="2024-06-23T17:59:15Z" level=info msg="Reconcile snapshotmoves3retry3-ldqpp" controller=dataupload dataupload=velero/snapshotmoves3retry3-ldqpp logSource="pkg/controller/data_upload_controller.go:115"
time="2024-06-23T17:59:15Z" level=warning msg="Pod is unschedulable 0/4 nodes are available: pod has unbound immediate PersistentVolumeClaims. preemption: 0/4 nodes are available: 4 No preemption victims found for incoming pod.." logSource="pkg/util/kube/pod.go:127" owner=snapshotmoves3retry3-ldqpp
time="2024-06-23T17:59:15Z" level=warning msg="Async fs backup data path canceled" dataupload=snapshotmoves3retry3-ldqpp logSource="pkg/controller/data_upload_controller.go:456"
time="2024-06-23T17:59:15Z" level=warning msg="failed to delete PV, for related PVC velero/snapshotmoves3retry3-ldqpp has no bind volume name" logSource="pkg/util/kube/pvc_pv.go:59"
time="2024-06-23T17:59:15Z" level=error msg="Cancel du velero/snapshotmoves3retry3-ldqpp because of expose error Pod is unschedulable: 0/4 nodes are available: pod has unbound immediate PersistentVolumeClaims. preemption: 0/4 nodes are available: 4 No preemption victims found for incoming pod.." controller=dataupload dataupload=velero/snapshotmoves3retry3-ldqpp logSource="pkg/controller/data_upload_controller.go:233"
time="2024-06-23T17:59:15Z" level=info msg="Reconcile snapshotmoves3retry3-ldqpp" controller=dataupload dataupload=velero/snapshotmoves3retry3-ldqpp logSource="pkg/controller/data_upload_controller.go:115"

Apparently the ceph-block storage class need some time before getting the pvc ready. Can we have some kind of timeout or wait awhile until pvc ready instead of a really quick tmeout ?

**Comment by Lyndon-Li** [comment]
See #7898

**Comment by sseago** [comment]
As menttioned above, this bug has been identified and fixed in the linked PR. Velero 1.14.1 will include the fix.

**Comment by DanielJuravski** [comment]
@sseago When 1.14.1 is planned to be released?

**Comment by sseago** [comment]
@DanielJuravski It should be out by the end of August.
